command/commands/user.py

```diff
@@ -43,7 +43,8 @@ def init_subparsers(self) -> _SubParsersAction:
         parser_view.set_defaults(which="view",
                                  help="View information about a given user.")
         parser_view. \
-            add_argument("--slack_id", type=str, action='store',
+            add_argument("--slack-id", metavar="SLACK-ID",
```

Comment:
What does `metavar` do?

Reply:
the metavar in this case represents the value that would be replaced in `/rocket user view --slack-id SLACK-ID`